# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/test/mochitest.ini
# Commit: c63ecdad6a3c
# Full Hash: c63ecdad6a3c60ba625517d34bb29c8aae9f5e95
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-02-22 17:34:52
# Regressor Bug: 1663931
# File Overlap Count: 1
# Description:
#   Bug 1663931 - Avoid moving focus when changing iframe remoteness. r=nika,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99215
# ==============================================================================

diff -r 828c62bfc0a2 -r c63ecdad6a3c dom/events/test/mochitest.ini
--- a/dom/events/test/mochitest.ini	Mon Feb 22 10:48:53 2021 +0000
+++ b/dom/events/test/mochitest.ini	Mon Feb 22 10:51:51 2021 +0000
@@ -46,7 +46,7 @@
 [test_bug402089.html]
 [test_bug405632.html]
 [test_bug409604.html]
-skip-if = toolkit == 'android' #TIMED_OUT
+skip-if = xorigin || toolkit == 'android' # xorigin: Bug 1682519, android: TIMED_OUT
 [test_bug412567.html]
 [test_bug418986-3.html]
 [test_bug422132.html]